edenai_apis/apis/groq/groq_api.py

```
@@ -0,0 +1,172 @@
+            stop_sequences=stop_sequences,
+            top_k=top_k,
+            top_p=top_p,
+            stream=stream,
+            response_format=response_format,
+            **kwargs,
+        )
+        return response
+
+    def llm__chat(
+        self,
+        messages: List = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)